lua/dap/breakpoints.lua

```diff
@@ -68,10 +68,17 @@ function M.toggle(opts, bufnr, lnum)
   if M.remove(bufnr, lnum) and not opts.replace then
     return
   end
+  local sign_name = 'DapBreakpoint'
+  if (non_empty(opts.condition)) then
+    sign_name = 'DapBreakpointCondition'
+  end
+  if (non_empty(opts.log_message)) then
+    sign_name = 'DapLogPoint'
+  end
```

Comment:
is that ok like this
or should the be a check to ensure that not both properties of the table are non empty

Reply:
The protocol specification allows both to be set - so this is fine. Or at least doesn't state explicitly that they shouldn't. So no reason to add artificial restrictions.